backend/messaging/message_bus.go

```diff
@@ -22,6 +24,24 @@ const (
 	TopicSubscriptions = "sensu:check"
 )
 
+// A Subscriber receives messages via a channel.
+type Subscriber interface {
+
+	// Receiver returns the channel a subscriber uses to receive messages.
+	Receiver() chan<- interface{}
```

Comment:
I'm about to kick myself in the face for asking this, but...

What if we did:
```
Receive(msg interface{})
```

And hid the receiving channel from the messagebus entirely.

Reply:
The problem with this is there's no way to prevent Receive from blocking; we'd have to start up goroutines.

Reply:
Ah right you are.